packages/ref-metrics-pmp/src/cmip_ref_metrics_pmp/pmp_driver.py

```diff
@@ -158,78 +151,33 @@ def build_pmp_command(  # noqa: PLR0913
     ----------
     driver_file
         Filename of the PMP driver script to run
-
-        Relative to the pcmdi_metrics package.
     parameter_file
         Filename of the parameter file to use
-
-        Relative to the cmip_ref_metrics_pmp.params package.
-    model_files
-        A list of model files to evaluate
-
-        Currently, we only support a single model file.
-    reference_name
-        Name of the reference dataset to use
-    reference_paths
-        Path to the reference dataset
-    source_id
-        Source ID of the model data
-    member_id
-        Member ID of the model data
-    output_directory_path
-        Output data where all the results are stored
     kwargs
         Additional arguments to pass to the driver script
     """
     # Note this uses the driver script from the REF env *not* the PMP conda env
     _driver_script = _get_resource("pcmdi_metrics", driver_file, use_resources=False)
     _parameter_file = _get_resource("cmip_ref_metrics_pmp.params", parameter_file, use_resources=True)
 
-    if isinstance(model_files, list):
-        modpath = " ".join([str(p) for p in model_files])
-    else:
-        modpath = model_files
-
-    if isinstance(reference_paths, list):
-        reference_data_path = " ".join([str(p) for p in reference_paths])
-    else:
-        reference_data_path = reference_paths
-
-    if len(model_files) != 1:
-        # Have some logic to replace the dates in the filename with a wildcard
-        raise NotImplementedError("Only one model file is supported at this time.")
-
     # Run the driver script inside the PMP conda environment
     cmd = [
         "python",
         _driver_script,
         "-p",
         _parameter_file,
-        "--modnames",
-        source_id,
-        "--exp",
-        experiment_id,
-        "--realization",
-        member_id,
-        "--modpath",
-        modpath,
-        "--reference_data_path",
-        reference_data_path,
-        "--reference_data_name",
-        reference_name,
-        "--results_dir",
-        output_directory_path,
-        "--cmec",
-        "--no_provenance",
     ]
 
     # Loop through additional arguments if they exist
     if kwargs:  # pragma: no cover
-        print("Additional info:")
         for key, value in kwargs.items():
-            print(f"  {key}: {value}")
-            cmd.extend([f"--{key}", str(value)])
+            if value:
+                cmd.extend([f"--{key}", str(value)])
+            else:
+                cmd.extend([f"--{key}"])
 
+    print("-- PMP command to run --")
     print("[PMP] Command to run:", " ".join(map(str, cmd)))
+    print("[PMP] Command generation for the driver completed.")
```

Comment:
Use logging routines instead of print so we can handle the logs correctly

```suggestion
    logger.info("-- PMP command to run --")
    logger.info("[PMP] Command to run:", " ".join(map(str, cmd)))
    logger.info("[PMP] Command generation for the driver completed.")
```

If `logger` isn't available locally, add `from loguru import logger`